### PR TITLE
Enable Kover for serialization, disable for Optics KSP

### PR DESCRIPTION
**Pull request by serras**
The Kover report is causing problems with the new version of KSP

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -68,10 +68,11 @@ dependencies {
   kover(projects.arrowContinuations)
   kover(projects.arrowCore)
   kover(projects.arrowCoreRetrofit)
+  // kover(projects.arrowCoreSerialization)
```
Comment (nomisRev) [inline build.gradle.kts:71]: Why is this commented?

File: build.gradle.kts [hunk 2]
```diff
@@ -68,10 +68,11 @@ dependencies {
...
   kover(projects.arrowFxCoroutines)
   kover(projects.arrowFxStm)
   kover(projects.arrowOptics)
-  kover(projects.arrowOpticsKspPlugin)
+  // kover(projects.arrowOpticsKspPlugin)
```
Comment (nomisRev) [inline build.gradle.kts:75]: I think this can be removed instead of comment?
```suggestion
```

File: build.gradle.kts (hunk 1 above)
Comment (serras) [reply build.gradle.kts:71]: I tried to add it for completeness, but it turns out that it causes problems with Kover (as the Optics KSP does). At the end I’ll remove both, and add a warning in the code to remember this.

File: build.gradle.kts (hunk 2 above)
Comment (serras) [reply build.gradle.kts:76]: I’d rather keep it alongside the comment in the corresponding section

**Review by nomisRev — APPROVED** [review]
Thanks @serras!

**Pull request merged**